hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenUpdateNftsHandlerTest.java

```diff
@@ -309,7 +379,7 @@ void failsEhenMetadatKeyNotSet() {
 
         Assertions.assertThatThrownBy(() -> subject.handle(handleContext))
                 .isInstanceOf(HandleException.class)
-                .has(responseCode(TOKEN_HAS_NO_METADATA_KEY));
+                .has(responseCode(INVALID_NFT_ID));
```

Comment:
This response code makes more sense for an NFT not found case, but was the handler logic intentionally changed to reflect this? Changing the expected response code here seems more like the result of an unintended change to which error is returned _first_

Reply:
The changeset makes this appear like the response code expected was changed however this test case was removed as the checks in handle were removed since it is checked in preHandle.